### ISSUE TITLE
Checksum mismatch

### ISSUE THREAD

**Issue by smackesey** [TileDB-Inc/homebrew-stable#17]
The most recent update gives a checksum mismatch:

```
Error: An exception occurred within a child process:
  ChecksumMismatchError: SHA256 mismatch
Expected: 10ac55dd764470884f642de817fc70c271006d53fdc404ed862079425eb66ebe
Actual: 20b1b4f97b9e40d36261982835f67f3d2f4f2c8c5d33b6b4709914e55049094d
```

**Comment by jakebolewski** [comment]
Whoops forgot to fix this after the draft release got finalized.

**Comment by jakebolewski** [comment]
closed by #18
